src/bin/pg_autoctl/primary_standby.c

```diff
@@ -80,14 +80,14 @@ GUC postgres_default_settings_13[] = {
 
 GUC citus_default_settings_pre_13[] = {
 	DEFAULT_GUC_SETTINGS_FOR_PG_AUTO_FAILOVER_PRE_13,
-	{ "shared_preload_libraries", "'citus','pg_stat_statements'" },
+	{ "shared_preload_libraries", "'citus,pg_stat_statements'" },
```

Comment:
When I  do
```
alter system set shared_preload_libraries TO 'citus,pg_stat_statements';
```
Postgres cannot start, complaining that the syntax is wrong.

It can  start with the following:
```
alter system set shared_preload_libraries TO citus,pg_stat_statements;
```

So, do we really need the single  quotes?

Reply:
I guess that ALTER SYSTEM command parsing does the list splitting in a different way than the configuration file parser. This fix comes from trying this code in a Citus environment.

Reply:
Nothing this as reference, you explanation sounds right. Because even if  I do:
```
alter system set shared_preload_libraries TO citus,pg_stat_statements;
```

On `postgresql.auto.conf`, it shows up:
```
shared_preload_libraries = 'citus, pg_stat_statements'
```